extension/lib/display.js

```diff
@@ -123,21 +117,15 @@ class Display {
     }
     
     /*
-    Updates reading timer based on containers after current tracker. 
+    Updates reading timer based on current sentence.
     */
-    updateTimer(readableDomEls, containerId) { // Call everytime you get to new paragraph
+    updateTimer(sentenceId) { // Call everytime the tracker moves.
```

Comment:
more granular estimation. display timer is based off current sentence, instead of container.
Also, we use doc's preprocessed .getNumWordsFromSentenceTilEnd, which will make this efficient.